controllers/serviceclaim_controller.go

```diff
@@ -301,6 +302,15 @@ func (r *ServiceClaimReconciler) processServiceClaim(
 	}
 
 	if !registeredServiceFound {
+		c := metav1.Condition{
```

Comment:
Let's create a method which updates the service claim passing in parameters like status,reason,message and state and update the status of service claim too.And use that wherever required.

Reply:
Here we are initializing a struct and calling a method. I don't know if it's a good idea to create function for the same.